server/app/tests/test.test.js

```diff
@@ -1,189 +1,62 @@
 import expect from 'expect';
 import request from 'supertest';
-
 import app from './../../app';
+import db from '../db/dbconnect';
 
-import {
-  menu,
-  Orders,
-} from '../db/dbconnect';
-
-describe('GET /api/v1/menu', () => {
-  it('should return menu of all avaliable food', (done) => {
-    request(app)
-      .get('/api/v1/menu')
-      .set('Accept', 'application/json')
-      .expect(200)
-      .expect((res) => {
-        expect(res.body.data === menu);
-        expect(200);
-      })
-      .end(done);
-  });
+before((done) => {
+  db.query('DELETE FROM users');
+  done();
 });
 
-describe('GET /api/v1/orders', () => {
-  it('respond with json containing a list of all orders', (done) => {
-    const data = {
-      foodId: 2,
-      quantity: 7,
+describe('GET /api/v1/auth/signup', () => {
+  it('should sign up a user when all the avaliable data is complete', (done) => {
+    const user = {
+      email: 'challenge@yahoo.com',
+      username: 'akpante',
+      password: '123456787',
+      address: '10adenekan fadeyi'
     };
-    Orders.push(data);
-    request(app)
-      .get('/api/v1/orders')
-      .set('Accept', 'application/json')
-      .expect(200)
-      .expect((res) => {
-        expect(res.body.data === Orders);
-        expect(200);
-      })
-      .end(done);
-  });
-});
-
-
-describe('POST /api/v1/orders', () => {
-  const data = {
-    foodId: 1,
-    quantity: 6,
-  };
-
-  it(`it should post order successfully when all
-   criterials are meet `, (done) => {
     request(app)
-      .post('/api/v1/orders')
-      .send(data)
+      .post('/api/v1/auth/signup')
+      .send(user)
       .expect(201)
-      .expect((res) => {
-        expect(res.body.status === 'success');
-        expect(200);
-      })
-      .end(done);
-  });
-
-  it('it should return a 404  when order is not found', (done) => {
-    const item = {
-      foodId: 700,
-      quantity: 7,
-    };
-    request(app)
-      .post('/api/v1/orders')
-      .send(item)
-      .set('Accept', 'application/json')
-      .expect(400)
-      .end(done);
-  });
-
-  it('it should not place an order when data is empty', (done) => {
-    request(app)
-      .post('/api/v1/orders')
-      .send({})
-      .expect(400)
-      .end(done);
-  });
-});
-
-describe('GET /api/v1/orders/:id', () => {
-  it('should fetch an order when the id is found', (done) => {
-    const order = {
-      food: 'meat pie',
-      id: 65,
-      foodId: 12,
-    };
-    Orders.push(order);
-    request(app)
-      .get('/api/v1/orders/65')
-      .set('Accept', 'application/json')
-      .expect(200)
-      .expect((res) => {
-        expect(res.body.status === 'success');
-      })
-      .end((err) => {
-        if (err) return done(err);
-        return done();
+      .expect()
```

Comment:
You are making no assertions in this test. Please fix.

Reply:
fixed